code/game/machinery/defibrillator_mount.dm

```diff
@@ -173,6 +174,7 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/machinery/defibrillator_mount, 28)
 	playsound(src, 'sound/items/deconstruct.ogg', 50, TRUE)
 	// Make sure processing ends before the defib is nulled
 	end_processing()
+	UnregisterSignal(defib, COMSIG_QDELETING)
```

Comment:
I notice the comment above this says `Make sure processing ends before the defib is nulled` which your comsig does the opposite, may cause a bug?